app/authentication/views.py

```diff
@@ -52,6 +55,33 @@ def _try_to_get_user(user_id):
         raise APIAuthUserDoesNotExist
 
 
+@api_view(["POST"])
+@permission_classes([IsDev, IsHS])
+def makeTIHLDEMember(request):
+    # Serialize data and check if valid
+    serializer = MakeUserSerializer(data=request.data)
+    if not serializer.is_valid():
+        return Response({"detail": "Ugyldig brukernavn"}, status=400)
+
+    # Get username and password
+    TIHLDE = Group.objects.get(slug=Groups.TIHLDE)
+    user_id = serializer.data["user_id"]
+    user = User.objects.get(user_id=user_id)
+    if user is not None:
+        try:
+            m = Membership.objects.get(user__user_id=user_id, group=TIHLDE)
```

Comment:
Kan vi bruke `get_or_create()` her for å slippe try-except klausen? Og så bruke det andre argumentet returnert for å avgjøre om den ble opprettet eller oppdatert.